harambe/cookies_handler.py

```diff
@@ -0,0 +1,8 @@
+from harambe.types import Cookie
+from datetime import datetime, timezone, timedelta
+
+
+def fix_cookie(cookie: Cookie) -> Cookie:
+    # Hardcode expiry to two days ahead
+    cookie["expires"] = (datetime.now(tz=timezone.utc) + timedelta(days=2)).timestamp()
+    return cookie
```

Comment:
we should test when the expiry is and only if its less than a day should we set it to a day ahead (instead of 2 days)

Reply:
yeah make sense